tests/fixture/tmpnet/network.go

```diff
@@ -84,9 +82,13 @@ func FindNextNetworkID(rootDir string) (uint32, string, error) {
 
 // Defines the configuration required for a tempoary network
 type Network struct {
-	NetworkConfig
 	NodeRuntimeConfig
 
+	Genesis       *genesis.UnparsedConfig
+	CChainConfig  FlagsMap
+	DefaultFlags  FlagsMap
```

Comment:
nit: can we find a better name for `DefaultFlags`? Are these `DefaultNetworkConfigs` or `DefaultGlobalConfigs` (they seems to hold both network and node configs)